opyn_test.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Created By: Anil@Opyn
+# Created Date: 06/07/2022
+# version ='0.1.0'
+# ---------------------------------------------------------------------------
+
+from opyn.definitions import *
+# from opyn.erc20 import ERC20Contract
```

Comment:
didn't yet get a chance, but I'll be also testing this locally at the next round